src/lib.rs

```diff
@@ -108,6 +108,32 @@ impl Utility {
             .build(ctx)
             .unwrap()
     }
+
+    /// Returns a random (x, y) tuple that is located within the specified width and height.
+    ///
+    /// The x value will be in the range [0, `width`), i.e. inclusive of 0 and exclusive of `width`.
+    /// The y value will be in the range [0, `height`), i.e. inclusive of 0 and exclusive of `height`.
+    ///
+    /// # Arguments
+    /// * `width` - The width of the area
+    /// * `height` - The height of the area
+    ///
+    /// # Example
+    ///
+    /// ```
+    /// # let mut utility = bbggez::Utility::new();
+    ///
+    /// let width = 100f32;
+    /// let height = 100f32;
+    /// let (x, y) = utility.random_location(width, height);
+    /// println!("X: {}, Y: {}", x, y);
+    /// ```
+    pub fn random_location(&mut self, width: f32, height: f32) -> (f32, f32) {
+        let area_x: f32 = self.rng.gen_range(0.0f32, width);
+        let area_y: f32 = self.rng.gen_range(0.0f32, height);
+
+        (area_x, area_y)
+    }
```

Comment:
I would like to return a Vector2 here. I want that as I almost always have my random locations be a vector2 for the vector math. 

Otherwise, looking great!